src/Pmi/Console/Command/DeployCommand.php

```diff
@@ -153,7 +153,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if ($this->local) {
-            $cmd = "php -S localhost:{$this->port} -t web/ web/local-router.php";
+            $cmd = "php -S 0.0.0.0:{$this->port} -t web/ web/local-router.php";
```

Comment:
Docker's port forwarding seems to require the server not be bound to `localhost` if you want it to be available on the host machine.